test/test_utils.jl

```diff
@@ -0,0 +1,48 @@
+using HolisticElectricityModel
+const hem = HolisticElectricityModel
+test_dir = dirname(@__FILE__)
+@testset "test read_param with county level index" begin
```

Comment:
Could use some more spacing? E.g., on either side of the line defining `test_dir`?